website/source/docs/platform/k8s/clients-outside-kubernetes.html.md

```diff
@@ -38,7 +38,7 @@ different pods to have different exposed ports.
 
 Consul typically requires a fully connected network. Therefore,
 nodes outside of Kubernetes joining a cluster running within Kubernetes must be able to communicate
-to pod IPs or Kubernetes node IPs via the network.
+to pod IPs or Kubernetes internal node IPs via the network.
```

Comment:
there's no way to expose the servers on internal node ips right now. I think you'd be best expanding this section to say that clients and servers must be addressable and that clients can be on pod ips or node ips depending on the value of the exposeGossipPorts command and that servers are only exposed on pod ips right now.

Reply:
Good point. I've updated that section. I've also removed the last part that talked about using `host_network` with the auto-join provider since it's not relevant if the servers can't be exposed.